packages/blade/src/components/Skeleton/Skeleton.stories.tsx

```diff
@@ -141,7 +141,7 @@ const BasicSkeleton = (): React.ReactElement => {
       width="100%"
       padding="spacing.5"
       borderRadius="medium"
-      backgroundColor="surface.background.level2.lowContrast"
+      backgroundColor="surface.background.gray.intense"
```

Comment:
These examples aren't really helpful when we press the show code button in docs to see the implementation.

<img width="1038" alt="Screenshot 2024-01-09 at 4 56 49 PM" src="https://github.com/razorpay/blade/assets/46647141/4c9ccd8e-e8c9-4fca-bcf0-dada6bac9899">

Ideally, we should display how we have built the shown layout but instead, it just shows `<BasicSkeleton />`.

Reply:
Not possible as of now. Storybook limitation.